concourse/pipeline.yml

```diff
@@ -30,6 +37,18 @@ resources:
         - concourse/Dockerfile
         - Gemfile*
 
+  - name: grafana-annotate-deploy
+    type: grafana-annotation
+    icon: chart-areaspline
+    source:
+      url: https://grafana-paas.cloudapps.digital
+      api_token: ((grafana-api-key))
+      tags:
+        - govuk-accounts
+        - account-manager-prototype
+        - deploy
+        - ((environment))
```

Comment:
Where does `((environment))` come from?

Reply:
Humm, that's a good point...
It was something they'd show [in the docs](https://github.com/alphagov/paas-grafana-annotation-resource#configuration), but now I think about it concourse won't have a sense of environment.
I can pass that as a param instead, will amend!